base/applications/cmdutils/fsutil/dirty.c

```diff
@@ -95,16 +95,16 @@ SetMain(int argc, const TCHAR *argv[])
 }
 
 static void
-PrintUsage(const TCHAR * Command)
+DirtyPrintUsage(const TCHAR * Command)
 {
-    PrintDefaultUsage(_T(" DIRTY "), Command, (HandlerItem *)&HandlersList,
-                      (sizeof(HandlersList) / sizeof(HandlersList[0])));
+    PrintDefaultUsage(_T(" DIRTY "), Command, (HandlerItem *)&DirtyHandlersList,
+                      (sizeof(DirtyHandlersList) / sizeof(DirtyHandlersList[0])));
```

Comment:
Nice chance to use `_countof` or `ARRAYSIZE` ?
other than that, lgtm including all fsutil changes.

Reply:
Applied. How does fsutil look now?